GitCommands/AsyncLoader.cs

```diff
@@ -151,7 +154,8 @@ public Task<T> Load<T>(Func<CancellationToken, T> loadContent, Action<T> onLoade
 
                     return loadContent(token);
                 }, token)
-                .ContinueWith((task) =>
+                .ContinueWith(
+                    (task) =>
             {
```

Comment:
This brace seems to be misaligned.

Reply:
Thanks, I'll get it fixed. I don't think there's a SCA rule for alignment of statement lambdas.

If this is merged, I'll configure R# to respect these code styles then run it on the whole solution and make another PR with any other inconsistencies like this.

Reply:
Fixed.